src/components/ActiveChart/index.js

```diff
@@ -53,7 +54,20 @@ export default class ActiveChart extends Component {
 
     return (
       <div className={styles.activeChart}>
-        <NumberInfo subTitle="目标评估" total="有望达到预期" />
+        <NumberInfo
+          subTitle={
+            <FormattedMessage
+              id="component.activeChart.target-assessment"
+              defaultMessage="Target assessment"
+            />
+          }
+          total={
+            <FormattedMessage
+              id="component.activeChart.expected-to-reach"
+              defaultMessage="Expected to reach"
+            />
```

Comment:
components should not use umi/plugin

Reply:
@chenshuai2144 so they cannot be localized? If so I will remove it